services/src/pro/server.rs

```diff
@@ -222,6 +222,7 @@ async fn start_postgres(
             .password(&db_config.password)
             .host(&db_config.host)
             .dbname(&db_config.database)
+            .port(db_config.port)
```

Comment:
can you also add this to the non-pro version: https://github.com/glombiewski/geoengine/blob/814d06609161cbdc75cc96bc3a58f1112d068630/services/src/server.rs#L53

Reply:
9cd7f7ac6e175ab4a0b9af46e88a8f798e7c8fc3